cylc/flow/scheduler.py

```diff
@@ -678,17 +678,15 @@ def _load_pool_from_point(self):
         released from runhead.)
 
         """
-        if self.config.start_point is not None:
-            start_type = "Warm" if self.options.startcp else "Cold"
-            LOG.info(f"{start_type} start from {self.config.start_point}")
+        start_type = "Cold"
+        if self.config.start_point > self.config.initial_point:
+            start_type = "Warm"
+        LOG.info(f"{start_type} start from {self.config.start_point}")
 
         flow_num = self.flow_mgr.get_new_flow(
             f"original flow from {self.config.start_point}"
         )
         for name in self.config.get_task_name_list():
-            if self.config.start_point is None:
-                # No start cycle point at which to load cycling tasks.
-                continue
```

Comment:
Pretty sure this condition is no longer possible. And L691 showing as never hit in code coverage: https://app.codecov.io/gh/cylc/cylc-flow/blob/master/cylc/flow/scheduler.py

Reply:
Checked in practice. Confirmed I can't get it to be true.